app/preprints/submit/controller.ts

```diff
@@ -0,0 +1,12 @@
+
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import Media from 'ember-responsive';
+
+export default class PreprintSubmitController extends Controller {
+    @service media!: Media;
+
+    get isMobile() {
+        return this.media.isMobile;
+    }
+}
```

Comment:
As @futa-ikeda mentioned, `is-mobile` is a builtin from `ember-responsive` now, so we don't need to wait for [his PR](https://github.com/CenterForOpenScience/ember-osf-web/pull/2153) to use `is-mobile` and remove this helper. We can do it here or use another PR to clean the code.